src/help/help.go

```diff
@@ -64,9 +68,123 @@ func help(topic string) string {
 	if f, present := m[topic]; present {
 		return helpFromBuildRule(f.FuncDef)
 	}
+
+	// Check plugins
+	if pluginHelp := helpForPlugin(topic); pluginHelp != "" {
+		return pluginHelp
+	}
+
+	return ""
+}
+
+// helpForPlugin returns some help text for a plugin
+func helpForPlugin(topic string) string {
+	config, err := core.ReadDefaultConfigFiles(nil)
+	if err != nil {
+		panic("Failed to read config")
+	}
+	if _, ok := config.Plugin[topic]; ok {
+		message := fmt.Sprintf("${BOLD_BLUE}%v${RESET} is a plugin defined in the ${GREEN}.plzconfig${RESET} file.\n", topic)
+
+		// This is if there's a subrepo specified for the plugin. If there isn't, we'll check for the subrepo in the plugins/
+		var subrepoName string
+		buildLabel := core.BuildLabel{Name: "all"}
+		if val, ok := config.Plugin[topic].ExtraValues["subrepo"]; ok {
```

Comment:
This should now be `config.Plugin[topic].Target` as per the latest master. This new `Target` field is also compulsory (as per https://github.com/thought-machine/please/commit/3d64e323719c5ad284655a2e11b42d404ea72f31#diff-bfaa51a2df379d9e9351315998d3720e62cd16b8ebd972e450e9508dcde68e9dR131), so it might simplify the logic here as well.